back/engines/commercial/idea_custom_fields/spec/acceptance/ideas_spec.rb

```diff
@@ -40,19 +40,34 @@
 
     context 'when the extra field is required' do
       let(:form) { create(:custom_form, :with_default_fields, participation_context: project) }
-      let!(:text_field) { create(:custom_field_extra_custom_form, key: extra_field_name, required: true, resource: form) }
 
       context 'when the field value is given' do
-        let(:custom_field_name1) { 'test value' }
-
+        # TODO: Refactoring
+        # - Validate input types in params + spec (can't create html_multiloc field in ideation or native survey)
         post 'web_api/v1/ideas' do
-          example_request 'Create an idea with an extra field' do
-            assert_status 201
-            json_response = json_parse(response_body)
-            idea_from_db = Idea.find(json_response[:data][:id])
-            expect(idea_from_db.custom_field_values.to_h).to eq({
-              extra_field_name => 'test value'
-            })
+          [
+            { factory: :custom_field_number, value: 42 },
+            { factory: :custom_field_linear_scale, value: 3 },
+            { factory: :custom_field_text, value: 'test value' },
+            { factory: :custom_field_multiline_text, value: 'test value' },
+            { factory: :custom_field_select, options: [:with_options], value: 'option1' },
+            { factory: :custom_field_multiselect, options: [:with_options], value: %w[option1 option2] },
+            { factory: :custom_field_multiselect_image, options: [:with_options], value: %w[image1] },
+            { factory: :custom_field_html_multiloc, value: { 'fr-FR' => '<p>test value</p>' } } # This field does not seem to be supported by ideation but very rarely occurs on production
```

Comment:
It seems that a extra custom html_multiloc field could be created by copying the built-in description field. So it might not have been enabled in the past.

Reply:
I'll update the comment as we know more about what happened now.